polygeist/test/polygeist-opt/parallellicm.mlir

```diff
@@ -1,12 +1,9 @@
-// RUN: polygeist-opt --parallel-licm --split-input-file --mlir-pass-statistics %s 2>&1 | FileCheck %s
+// RUN: polygeist-opt --parallel-licm --split-input-file %s 2>&1 | FileCheck %s
```

Comment:
likely accidentally reverted the stat change in `parllellicm.mlir`?

Reply:
No I did it intentionally because I found out that the statistics are not emitted per function. In a future PR we can add a separate test just to check the statistic report (if you want).